scripts/opensearch-build/release.jenkinsfile

```diff
@@ -0,0 +1,104 @@
+properties([
+  parameters([
+    string(
+        name: 'DistManifestUrl',
+        description: 'The url to distribution manifest for this release')])
+])
+
+lib = library(
+    identifier: 'jenkins@uni-poc',
+    retriever:modernSCM(
+        [$class: 'GitSCMSource',
+        remote: 'https://github.com/peternied/opensearch-build.git',
+        traits: [gitBranchDiscovery()]
+        ])
+    )
+
+Map config = [
+    cleanup: { -> echo "Totally unneed clean for demo purposes"}
+]
+
+standardReleasePipeline(config) {
+    def distManifest = new FakeDistManifest(fakeYmlFromUrl(params.DistManifestUrl))
+
+    def downloadPath = fakeDownloadArtifacts(distManifest)
+
+    fakeSignArtifacts(
+        artifactsPath: downloadPath,
+        signatureType: '.sig',
+        distributionPlatform: 'linux',
+    )
+    
+    fakePublishToMaven(
+        mavenArtifactsPath: "${downloadPath}/maven",
+        autoPublish: true
+    )
+
+    fakePublishToArtifactRepositry(
```

Comment:
Are these the .tar.gz files?

Reply:
Yes - they are described in the manifest that I hand-waved over